src/algebra/category/CommRing/limits.lean

```diff
@@ -13,6 +13,11 @@ import ring_theory.subring
 
 Further, these limits are preserved by the forgetful functor --- that is,
 the underlying types are just the limits in the category of types.
+
+## Implementation note
+
+Lean suffers in some unification steps, and has to do a lot of unfolding to check that types match.
+We use a hack that prefixing such proofs with `by convert` can give a huge speedup.
```

Comment:
Perhaps this should be a library note that we can use to tag otherwise strange-looking `by convert` proofs.

Reply:
Done.